site/_js/web-components/enhanced-select.js

```diff
@@ -0,0 +1,424 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Progressively enhances the select element. Supports
+ * both single and multi select.
+ */
+import {BaseElement} from './base-element';
+import {html} from 'lit-element';
+import {unsafeSVG} from 'lit-html/directives/unsafe-svg';
+import arrowDownIcon from '../../_includes/icons/arrow-down.svg';
+import {generateIdSalt} from '../utils/salt';
+import closeIcon from '../../_includes/icons/close.svg';
+
+const keyReg = new RegExp('^(Key|Digit|Numpad)', 'i');
+
+export class EnhancedSelect extends BaseElement {
+  static get formAssociated() {
+    return true;
+  }
+
+  constructor() {
+    super();
+
+    // @ts-ignore
```

Comment:
What are exactly you ignoring here?

Reply:
I'm getting an erroneous  'property does not exist' error without the ignore statement...